collection+xattrs.go

```diff
@@ -618,16 +619,74 @@ func removeUserXattrs(xattrs semiParsedXattrs) {
 	}
 }
 
+// macroExpand takes the macro and assigned 'value' the correct value based on what the macro is provided (this is gocb macro format)
+func (e *event) macroExpand(value string, casServerFormat string, bucketUUID string) string {
+	if value == "\"${Mutation.CAS}\"" {
```

Comment:
I suggest making this a `switch` statement, and using back-quotes around the strings to avoid having to escape double-quotes.